Umbraco-Cloud/Set-Up/Usage/index.md

```diff
@@ -0,0 +1,23 @@
+---
+versionFrom: 7.0.0
+---
+
+# Usage on your Umbraco Cloud project
+
+In the Umbraco Cloud settings tab, we recently introduced a new setting called Usage.
```

Comment:
It won't be a new page in half a year 😉 
Perhaps consider not calling it *new*.

Reply:
Thanks i have now changed it a bit 😄